client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -764,20 +762,11 @@ export const PlanComparisonGrid: React.FC< PlanComparisonGridProps > = ( {
 			<PlanComparisonHeader className="wp-brand-font">
 				{ translate( 'Compare our plans and find yours' ) }
 			</PlanComparisonHeader>
-			<PlanTypeSelector
+			<TermExperimentPlanTypeSelector
+				isEligible={ true }
```

Comment:
If possible, it would be prudent to put `is2023PricingGridVisible` in here as well.

Reply:
Good point. However, I didn't put it in because this component is only used in the 2023 pricing grid, and that's also why that property isn't available here. If I added it, it would be removed in the near future when we either concluding this experiment or exposing this new pricing grid to all the flows, so I chose to not. Does that make sense to you?